packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -23,6 +23,7 @@ export function createKusama (t: TFunction): EndpointOption {
       Parity: 'wss://kusama-rpc.polkadot.io',
       OnFinality: 'wss://kusama.api.onfinality.io/public-ws',
       'Patract Elara': 'wss://pub.elara.patract.io/kusama',
+      '🛡 DWELLIR 🛡': 'wss://kusama-rpc.dwellir.com:9977',
```

Comment:
```suggestion
      DWELLIR: 'wss://kusama-rpc.dwellir.com:9977',
```

Reply:
Ok! We will change the ports to standard web ports as well. Some firewalls might block these ports.. Update will come.